demo/jsf-flows-portlet/src/main/webapp/booking/flight.xhtml

```diff
@@ -2,25 +2,25 @@
 <ui:composition xmlns="http://www.w3.org/1999/xhtml" xmlns:alloy="http://liferay.com/faces/alloy"
 	xmlns:f="http://xmlns.jcp.org/jsf/core" xmlns:h="http://xmlns.jcp.org/jsf/html" xmlns:jsf="http://xmlns.jcp.org/jsf"
 	xmlns:ui="http://xmlns.jcp.org/jsf/facelets">
-	<alloy:field id="departureField" label="#{i18n['departure-airport']}" required="true">
-		<h:selectOneMenu id="departureId" value="#{bookingFlowModelBean.bookingDepartureId}">
+	<alloy:field id="departureField" label="#{i18n['departure-airport']}">
+		<h:selectOneMenu id="departureId" value="#{bookingFlowModelBean.bookingDepartureId}" required="true">
 			<f:selectItem itemLabel="#{i18n['select']}" itemValue="" />
 			<f:selectItems itemValue="#{airport.airportId}" itemLabel="#{airport.city}: #{airport.name}"
 				value="#{airportService.airports}" var="airport" />
 			<f:ajax render="departureField" />
 		</h:selectOneMenu>
 		<h:message for="departureId" />
 	</alloy:field>
-	<alloy:field id="arrivalField" label="#{i18n['arrival-airport']}" required="true">
-		<h:selectOneMenu id="arrivalId" value="#{bookingFlowModelBean.bookingArrivalId}">
+	<alloy:field id="arrivalField" label="#{i18n['arrival-airport']}">
+		<h:selectOneMenu id="arrivalId" value="#{bookingFlowModelBean.bookingArrivalId}" required="true">
 			<f:selectItem itemLabel="#{i18n['select']}" itemValue="" />
 			<f:selectItems itemValue="#{airport.airportId}" itemLabel="#{airport.city}: #{airport.name}"
 				value="#{airportService.airports}" var="airport" />
 			<f:ajax render="arrivalField" />
 		</h:selectOneMenu>
```

Comment:
@pcwhite, please add a message for the arrival selector here. Also please make sure that the submit button for this form renders the whole form.